imports/api/playlists/server/publications.js

```diff
@@ -1,27 +1,32 @@
 import { Meteor } from 'meteor/meteor';
 import { Playlists } from '../playlists_collection.js';
-import { HTTP } from 'meteor/http';
-import { Random } from 'meteor/random';
 import { NowPlaying } from '../now_playing.js';
+import { pagination } from 'meteor/kurounin:pagination';
 
 Meteor.publish('playlist', function (id) {
   check(id, Number);
   return Playlists.find({ spinPlaylistId: id });
 });
 
-Meteor.publish('playlists', function(options) {
+Meteor.publish('playlists0', function(options) {
   check(options, {
     sort: Object,
     limit: Number
   });
   return Playlists.find({}, options);
 });
 
+Meteor.publish('allPlaylists', function() {
+  return Playlists.find();
+});
```

Comment:
These kinds of publications could lead to (potentially) really big transfers of data when the number of playlists grows high. These days I always try to limit publications, either in number of entries returned, or in which fields are included/excluded, or both. Not sure if that's possible here or not. Just pointing it  out.